src/mn_setup.c

```diff
@@ -3830,16 +3830,16 @@ boolean MN_SetupResponder(menu_action_t action, int ch)
        current_item->m_flags |= S_HILITE;
     }
 
-    if ((current_item->m_flags & S_FUNC) && action == MENU_ENTER)
+    if ((current_item->m_flags & S_CALIBR) && action == MENU_ENTER)
     {
         if (ItemDisabled(current_item->m_flags))
         {
             M_StartSound(sfx_oof);
             return true;
         }
-        else if (current_item->var.func)
+        else
```

Comment:
Do we need to check for `current_item->action` here?

Reply:
No, because we only have one `S_CALIBR`, which always has `action`.